### PR TITLE
Lj 24 layout -  Draft  Work in progress 

### PR DESCRIPTION
**Pull request by MarcoGoC**
# Branch wont be updated, 
## instead the changes will be break down into 3 separated PR with different code 

[LJ-24] changes are getting too big splitting the changes in two PR 

 Added basic barebone language translation 
 Added Meta, Header, Footer from React based Prototype project
 Updated the components to work with Next 
 Updated links and translations

## Test Instructions

1. grab the branch 
1. do npm i 
1. npm start dev
1. check the browser

## Meets Definition of Done

- [ ] TODO: Fonts, margins, canada logo, change label to 'english' after translation along with gov logo

[LJ-24]: https://lj-decd.atlassian.net/browse/LJ-24

### CODE REVIEW

**Review by P0NDER0SA — APPROVED** [review]
This looks pretty clean @MarcoGoC .  I'm going to add the parameter to --passWithNoTests to pass this until we actually write a test :)

**Review by pcraig3 — CHANGES_REQUESTED** [review]
There’s quite a few issues to fix up here.

I’ve commented about most of them, but there are probably more things that will come up.

Mostly though we should avoid ~1000 line PRs like this. There are multiple stories being addressed in this PR and it will probably get bigger rather than smaller. It would have been better to split this into multiple PRs — eg, doing the header first, and the footer later, and then the translations afterwards. 

It’s really hard to review PRs like this and it will make it harder for others to do PRs alongside it as there is a large possibility of conflicts later on.

File: components/atoms/ActionButton.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+/**
+ * Button component which is used to trigger actions on the page
+ */
+export function ActionButton(props) {
```
[the remark this answers is not in the document]
Comment (MarcoGoC) [reply components/atoms/ActionButton.js:7]: yes, actually the alpha site took it from the prototype

**Comment by MarcoGoC** [comment]
Branch wont be updated 
Leaving here for comments

**Pull request closed without merging**